card/mod/06_bootstrap/lib/javascript/bootstrap_modal_wagn.js

```diff
@@ -0,0 +1,27 @@
+wagn.slotReady(function(slot) {
+  slot.find('.modal.fade').on('loaded.bs.modal', function(e) {
+    $(this).trigger('slotReady');
+  });
+  slot.find('[data-toggle=\'modal\']').off("click").on('click', function(e) {
```

Comment:
The `off("click")` can remove all binded handler from bootstrap. We need to override the onclick handler on all `'[data-toggle=\'modal\']'` to prevent bootstrap from getting content with `load`.